packages/network/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@shopify/network",
+  "version": "0.0.0",
+  "license": "MIT",
+  "description": "Common values related to dealing with the network",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "tsc --p tsconfig.build.json",
+    "prepublishOnly": "yarn run build"
+  },
+  "publishConfig": {
+    "access": "public",
+    "@shopify:registry": "https://registry.npmjs.org"
+  },
+  "author": "Shopify Inc.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Shopify/quilt.git"
+  },
+  "bugs": {
+    "url": "https://github.com/shopify/quilt/issues"
+  },
+  "homepage": "https://github.com/Shopify/quilt/blob/master/packages/network/README.md",
+  "dependencies": {},
+  "devDependencies": {
+    "typescript": "~3.0.1"
+  },
+  "files": ["dist/*"]
```

Comment:
Nitpick: in the other package.json, this was formatted with line breaks around the square braces.

Reply:
This is just Yarn formatting the one I added a dependency to, I'd rather leave alone until yarn is needed and will auto-format it.